modules/lang-painless/src/main/java/org/elasticsearch/painless/symbol/SemanticScope.java

```diff
@@ -7,7 +7,7 @@
  * not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *    http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
?

Reply:
Sigh. Intellij is misbehaving again. Fixed.